source/elements/oneDAL/source/common_interface/error_handling.rst

```diff
@@ -1,3 +1,127 @@
+.. _error_handling:
+
 ==============
 Error handling
 ==============
+
+oneDAL error handling relies on mechanism of C++ exceptions. If an error occurs,
+it must be propagated at the point of a function call where it can be caught
+using С++ exception handlers.
+
+Exceptions classification
+=========================
+
+Classification is aligned with C++ STL classification. oneDAL base exception
+classes are abstract. Leaf classes are also inherited from the corresponding STL
+exceptions. Only these classes can be thrown.
+
+The root of exception hierarchy is ``onedal::exception``. All throwing
```

Comment:
The base class of the exception hierarchy is ...The rest exception classes are derived from this class.